pkg/collect/longhorn.go

```diff
@@ -66,13 +66,7 @@ func (c *CollectLonghorn) Collect(progressChan chan<- interface{}) (CollectorRes
 	// collect nodes.longhorn.io
 	nodes, err := client.Nodes(ns).List(ctx, metav1.ListOptions{})
 	if err != nil {
-
-		v := viper.GetViper()
-		if v.GetBool("debug") {
-			return nil, errors.Wrap(err, "list nodes.longhorn.io")
-		}
-
-		return output, nil
+		return nil, &types.NotFoundError{Name: "list nodes.longhorn.io"}
```

Comment:
Same here. Just log the message using `logger.Printf`.

Only return the `NofFoundError` if the function is used in multiple places and callers need to differentiate errors. This code here is collector logic which the collector will dictate on what to do with certain errors

Reply:
changed.
However, when I try to write the test case for longhorn and ceph collect, I didn't any test function for ceph or longhorn Collect function. So I skipped them. I have added a test for ceph anazlyer.